outputs.tf

```diff
@@ -45,3 +45,8 @@ output "public_subnets_route_table" {
 output "private_subnets_route_table" {
   value = "${element(concat(aws_route_table.private_routetable.*.id, list("")), 0)}"
 }
+
+output "test" {
```

Comment:
Remove test code or rename to something more meaningful please :)

Reply:
fixed